pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -1071,6 +1071,9 @@ public static class Offline {
      */
     public static final String SEGMENT_UPLOAD_START_TIME = "segment.upload.start.time";
 
+    public static final String CONFIG_SEGMENT_SERVER_UPLOAD_TO_DEEP_STORE = "segment.server.upload.to.deep.store";
```

Comment:
We don't usually configure this constant here. See `HelixInstanceDataManagerConfig` for example

Reply:
Fixed.